src/Apps/Purchase/PurchaseApp.tsx

```diff
@@ -56,9 +59,11 @@ const OrderRow = (props: OrderRowProps) => {
   )
 }
 
-export class PurchaseApp extends React.Component<Props, {}> {
-  render() {
-    const { orders } = this.props
+export const PurchaseApp = (props: Props) => {
+  const { orders } = props
+  const { user } = useContext(SystemContext)
+  const isAdmin = userIsAdmin(user)
```

Comment:
Does `/purchases` have a top-level force route? If so, this should go there at the express level.